### PR TITLE
updated homepage copy layout

### PR DESCRIPTION
**Pull request by ThePeach**
- added has-centered-text utility-class
- added atom list-two-columns
- updated homepage layout and organisation

### CODE REVIEW

File: src/ui-lib/sass/06-utilities/_has-centered-text.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.grav-u-has-centered-text {
```
Comment (franz899) [inline src/ui-lib/sass/06-utilities/_has-centered-text.scss]: This is an interesting name for a css class? Isn't this naming convention usually used for method that return a boolean?

File: src/ui-lib/sass/gravity.scss [hunk 2]
```diff
@@ -81,3 +82,4 @@
 // === Utilities layer ===
 // Utilities and helper classes with ability to override anything which goes before in the
 // triangle, eg. hide helper class.
+@import '06-utilities/has-centered-text';
```
Comment (franz899) [inline src/ui-lib/sass/gravity.scss]: Should we have utility classes grouped by functionality? This one is a very specific file name.

Comment (ThePeach) [reply src/ui-lib/sass/gravity.scss]: fair point, I'll get it a generic `text` file for now

File: src/styleguide/_patterns/03-organisms/00-page-structure/03-section.mustache [hunk 3]
```diff
@@ -7,12 +7,12 @@
     {{#blockquote}}<blockquote>{{{blockquote}}}</blockquote>{{/blockquote}}
     {{#inlineRow}}{{>organisms-list-inline-row}}{{/inlineRow}}
     {{#imgCards}}{{>organisms-list-img-cards}}{{/imgCards}}
-    {{#orderedList}}
-    <ul>
+    {{#twoColumnsList}}
```
[the remark this answers is not in the document]
Comment (ThePeach) [reply src/styleguide/_patterns/03-organisms/00-page-structure/03-section.mustache]: it wasn't an ordered list in the first place 🤦‍♂️  it made more sense to have a more generic variable name that controls it.

**Pull request merged**